elastic/kibana
# [maps] fix Air-gapped enviroment hitting 400 error loading fonts for layer

Closes https://github.com/elastic/kibana/issues/165974

The root cause of the problem is calling `basePath.prepend` on a path without a leading `/`. Maps is not consistent with path constants. Some have leading `/` while others do not. To resolve the issue, this PR updates all path constants to consistently have leading `/`.

PR adds functional test runner with `map.includeElasticMapsService: false` to ensure maps is tested without EMS enabled to catch future regressions.

### Test instructions
* set *map.includeElasticMapsService: false* in kibana.dev.yml
* install sample data set
* create new map with documents layer
* Configure label styling
* Verify layer is displayed with labels

<img width="500" alt="Screen Shot 2023-09-13 at 6 26 23 AM" src="https://github.com/elastic/kibana/assets/373691/44d12e87-9b80-424c-9bc9-126b373bdf18">
